base/_mixins.scss

```diff
@@ -1,50 +1,100 @@
-//   color: red;
-// }
-// .no-js .link__disabled, .no-js .link__disabled:link, .no-js .link__disabled:visited, .no-js .link__disabled:hover, .no-js .link__disabled:focus, .js.no-touchevents .link__disabled, .js.no-touchevents .link__disabled:link, .js.no-touchevents .link__disabled:visited, .js.no-touchevents .link__disabled:hover, .js.no-touchevents .link__disabled:focus, .js.touchevents .link__disabled, .js.touchevents .link__disabled:active {
-//   cursor: default;
-//   color: silver;
-// }
+
+/// touch-hover() =
+/// Provide an accesible hover for non-touch devices, turns it into the active state for mobile, and maintains a no-js fallback
+/// @author jhogue
+/// @require Modernizr.js to get .no-touchevents classes
+/// @content the @content directive is used
+// 
+/// @example
+///   //  .link {
+///   //   padding: .5em 1em; // Styles that do not change when interacting with this element
+///   //  
+///   //   @include touch-hover('idle') {
+///   //     color: black; // Styles for the :link and :visited states
+///   //   }
+///   //   @include touch-hover('hover') {
+///   //     color: red; // Styles for the :focus, :hover and :active states
+///   //   }
+///   //  
+///   //   &__disabled {
+///   //     @include touch-hover('hover', true) {
+///   //      color: silver; // All psuedo states are included
+///   //     }
+///   //   }
+///   //  }
+///
+/// @output
+///   //  .link {
+///   //    padding: .5em 1em;
+///   //  }
+///   //  .link,
+///   //  .link:link,
+///   //  .link:visited {
+///   //    color: black;
+///   //  }
+///   //  .no-js .link:hover,
+///   //  .no-js .link:focus,
+///   //  .js.no-touchevents .link:hover,
+///   //  .js.no-touchevents .link:focus,
+///   //  .js.touchevents .link:active {
+///   //    color: red;
+///   //  }
+///   //  .no-js .link__disabled,
+///   //  .no-js .link__disabled:link,
+///   //  .no-js .link__disabled:visited,
+///   //  .no-js .link__disabled:hover,
+///   //  .no-js .link__disabled:focus,
+///   //  .js.no-touchevents .link__disabled,
+///   //  .js.no-touchevents .link__disabled:link,
+///   //  .js.no-touchevents .link__disabled:visited,
+///   //  .js.no-touchevents .link__disabled:hover,
+///   //  .js.no-touchevents .link__disabled:focus,
+///   //  .js.touchevents .link__disabled,
+///   //  .js.touchevents .link__disabled:active {
+///   //    cursor: default;
+///   //    color: silver;
+///   //  }
+///
+// @parameter 
+// 1. 'idle' or 'hover'
+// 2. true/false
+// 
+// @options
+// 1. Set the name of the class hooks that moderniizr provides
+// 2. Turn off modernizr as a dependency
 //
 @mixin touch-hover( $state: 'idle', $disabled: false ) {
 	$modernizr-hook-off: 'no-touchevents';
 	$modernizr-hook-on: 'touchevents';
 	$modernizr: true !default; // set this to false to maintain mixin but use the no-modernizr fallbacks
 
+  @if $state != 'idle' or $state != 'hover' {
```

Comment:
Hmm, DOES NOT WORK. SASS will not compile if I use the not keyword… the statement is never true, so because of the @error directive, the compilation stops.